src/handler.cr

```diff
@@ -79,21 +89,20 @@ class KemalJWTAuth::Handler < Kemal::Handler
 
   private def decode(jwt) : {UserHash, Hash(String, JSON::Any)}
     payload, header = JWT.decode token: jwt, key: @secret_key, algorithm: @algorithm
-    pp! payload, header
-    uh = UserHash.new payload.size
-    payload.each { |k, v| uh[k] = v.as_s? || v.as_i? || v.as_bool? }
-    {uh, header}
+    user_hash = UserHash.new payload.size
+    payload.each { |k, v| user_hash[k] = v.as_s? || v.as_i? || v.as_bool? }
+    {user_hash, header}
   end
 
-  macro send_json(data)
+  private def send_json(context, data) : HTTP::Server::Context
```

Comment:
You're right...making this a macro is an unnecessary obfuscation. However, "send JSON context" doesn't seem like the right phrasing. How about this?
```crystal
private def send_json(data, to context : HTTP::Server::Context) : HTTP::Server::Context
```